src/bespokelabs/curator/request_processor/openai_request_mixin.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any
+from bespokelabs.curator.types.generic_request import GenericRequest
+
+
+class OpenAIRequestMixin:
+    """Mixin class for creating OpenAI-specific API requests.
+
+    Provides shared functionality for both batch and online OpenAI request processors.
+    """
+
+    def create_api_specific_request_online(self, generic_request: GenericRequest) -> dict:
```

Comment:
Do we need this `_online` suffix?

Reply:
yes since the request has additional parameters for batch, which is defined in `create_api_specific_request_batch`